.env-cmdrc

```diff
@@ -0,0 +1,16 @@
+{
+  "dev:b2c": {
```

Comment:
What's the benefit of grouping it like this vs having 4 environments: `dev, ci, b2b, b2c` and composing them?

Reply:
That we need combinations (different URLs for local and CI, set the B2B var to true). We don't have a `b2b` env alone.

Reply:
But this package allows to use different environments at the same time.

eg: `env-cmd ci,b2b <ommand>`

We have fallbacks in our environment.ts file and also composing will remove some duplication from this file. We would have single url for ci, etc.

Reply:
the idea for the `dev` section is that devs can tweak it the way they want. For CI, I still think we need 2. Maybe we can discuss?

Reply:
Yep. Let's discuss